Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -882,3 +882,7 @@
 #define ENABLE_WEBGL_IN_WORKERS 1
 #endif
 
+#if !defined(ENABLE_PURGEABLE_DECODED_IMAGE_DATA_CACHE)
+#define ENABLE_PURGEABLE_DECODED_IMAGE_DATA_CACHE 1
+#endif
+
```

Comment:
Because I haven't implemented the other platform backends for SharedMemory::setVolatile/setNonVolatile. It's probably doable but I'm less familiar with the other platforms.